lld/COFF/Driver.cpp

```diff
@@ -2357,33 +2359,32 @@ void LinkerDriver::linkerMain(ArrayRef<const char *> argsArr) {
   }
 
   // Handle /entry and /dll
-  {
+  ctx.forEachSymtab([&](SymbolTable &symtab) {
     llvm::TimeTraceScope timeScope("Entry point");
```

Comment:
Hmm, how does this `TimeTraceScope` work if targeting arm64x so this is ran twice?

Reply:
I think it will sum up times from each execution [here](https://github.com/llvm/llvm-project/blob/c805df69994a1f1e8f095b690c9af9e9c463906b/llvm/lib/Support/TimeProfiler.cpp#L205). I will keep the extra scope if you prefer.

Reply:
Nah, if it works as expected while entering the same `TimeTraceScope` multiple times, it's totally fine for me to keep it like this, avoiding another indentation level.